docs/use_cases/UI-client-credentials/README.md

```diff
@@ -0,0 +1,78 @@
+# Web UI with Client Credentials
+
+## Introduction
+
+Example of Web GUI (Graphic User Interface) application on Python ([Flask](https://flask.palletsprojects.com/en/stable/)) to interface HCP services using [pyFirecREST](https://pyfirecrest.readthedocs.io/en/stable/).
+
+This example uses Authentication with an Identity Provider (IdP) with the [Client Credential Workflow](https://oauth.net/2/grant-types/client-credentials/).
+
+Follow the Configuration guide to adapt your client credentials to the app.
+
+## Prerequisites
+
+- Docker installed
+- Knowledge of Python
+
+## Configuration
+
+1. Copy the configuration file (`src/config.py.orig`) and rename it as `src/config.py`
+
+    !!! example "Copy empty configuration file"
+        ```bash
+        $ cd src
+        $ cp config.py.orig config.py
+        ```
+
+2. Replace the follwing values with the specifics for this training 
```

Comment:
```suggestion
2. Replace the following values with the specifics for this training 
```

Reply:
Not sure it matters, but is "for this training" accurate? I would say for this tutorial, but I don't have a strong opinion.

Reply:
Yes, that's a leftover from a past training. "Tutorial" is the correct wording. Thanks.